### PR TITLE
New package: activate-linux-0.0.2

### PR DESCRIPTION
**Pull request by nadevko**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### New package
- This new package conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements): **YES**

#### Local build testing
- I built this PR locally for my native architecture, x86_64

### CODE REVIEW

File: srcpkgs/activate-linux/template [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Template file for 'activate-linux'
+pkgname=activate-linux
+version=0.0.1
+revision=1
+hostmakedepends="libXfixes-devel libX11-devel libXinerama-devel cairo-devel"
+short_desc="\"Activate Windows\" watermark ported to Linux"
+maintainer="Nadeŭka <nadevko@riseup.net>"
+license=custom:based-license-1.4.1
```
Comment (Chocimier) [inline srcpkgs/activate-linux/template:8]: License is nonfree, please mark as such. Different version is used here.

**Comment by nadevko** [comment]
https://github.com/MrGlockenspiel/activate-linux/issues/106

**Comment by Chocimier** [comment]
- `*-devel` should be in makedepends, not hostmakedepends to enable cross compilation
- expand `$homepage`
- please quote homepage that xnew quotes: license, homepage, distfiles

File: srcpkgs/activate-linux/template [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# Template file for 'activate-linux'
+pkgname=activate-linux
+version=0.0.2
+revision=1
+build_style=gnu-makefile
+hostmakedepends="libXfixes-devel libX11-devel libXinerama-devel cairo-devel"
+makedepends=pkg-config
+short_desc="\"Activate Windows\" watermark ported to Linux"
+maintainer="Nadeŭka <nadevko@riseup.net>"
+license=GPL-3.0-or-later
```
Comment (Chocimier) [inline srcpkgs/activate-linux/template:10]: GPL-3.0-only

File: srcpkgs/activate-linux/template [hunk 3]
```diff
@@ -0,0 +1,20 @@
+# Template file for 'activate-linux'
+pkgname=activate-linux
+version=0.0.2
+revision=1
+build_style=gnu-makefile
+hostmakedepends="libXfixes-devel libX11-devel libXinerama-devel cairo-devel"
```
Comment (abenson) [inline srcpkgs/activate-linux/template:6]: Swap `hostmakedepends` and `makedepends`.

**Pull request closed without merging**